src/psyclone/psyir/nodes/node.py

```diff
@@ -1077,6 +1077,8 @@ def ancestor(self, my_type, excluding=None, include_self=False,
                                   search.
         :param limit: an optional node at which to stop the search.
         :type limit: Optional[:py:class:`psyclone.psyir.nodes.Node`]
+        :param shared_with: an optional node which must also have the
```

Comment:
Please also mention this functionality in the main part of the docstring (L1070).

Reply:
Added